mobify/progressive-web-scaffold
# Bug/remove alert confirmation

Triggers two astro events when the confirmation page is rendered

 **JIRA**: https://mobify.atlassian.net/browse/HYB-1025

Related PRs: https://github.com/mobify/progressive-web-scaffold/pull/398

## Changes
- Trigger disabling of the checkout alert
- Trigger cart-updating to have zero products

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Open in app, check out and click the "close" button
